components/toaster.tsx

```diff
@@ -11,18 +10,33 @@ export default function Toaster() {
     dispatch(removeToast(i));
   }
 
+  useEffect(() => {
```

Comment:
I forgot to add the `delay={3000}` prop to the `Toast` component. all we need to is add that prop and `onClose={() => dispatch(removeToast(i))}`